src/NUnitFramework/framework/Constraints/Comparers/EquatablesComparer.cs

```diff
@@ -73,8 +73,6 @@ private static MethodInfo FirstImplementsIEquatableOfSecond(Type first, Type sec
 
         private static IList<KeyValuePair<Type, MethodInfo>> GetEquatableGenericArguments(Type type)
         {
-            // NOTE: Original implementation used Array.ConvertAll and
-            // Array.FindAll, which don't exist in the compact framework.
```

Comment:
I originally left this in just to remind us in case we wanted to go back to the original implementation. Just saying... I'm OK with removing it.

Reply:
:+1: I like this more verbose code because perf is better without using delegates.